components/GreetingButton.js

```diff
@@ -0,0 +1,31 @@
+import { View, Text, StyleSheet } from 'react-native';
+
+const GreetingButton = () => {
+    return (
+        <View style={styles.greetingContainer}>
+            <Text style={styles.title}> Happiness is the highest form of health. </Text>
+        </View>
+    )
+}
+
+export default GreetingButton;
+
+const styles = StyleSheet.create({
+    greetingContainer: {
+        backgroundColor: '#869684',
+        height: 60,
+        width: 300, 
+        alignItems: 'center',
+        justifyContent: 'center',
+        borderWidth: 1,
+            borderColor: '#3A376F',
+            borderRadius: 18,
+    },
+
+    title: {
+        // fontFamily: ??? needs to be imported
+        fontSize: 20,
```

Comment:
Great job getting the font style to fit inside the box.
Also, curious about importing fonts.